packages/react-core/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Icon/icon';
+import { css } from '@patternfly/react-styles';
+import { Spinner } from '../Spinner';
+
+export interface IconComponentProps extends Omit<React.HTMLProps<HTMLSpanElement>, 'size'> {
+  /** Icon content when isInProgress is set to false */
```

Comment:
Do we need to mention isInProgress here?  The description kind of  makes it seem like the whole purpose of the component is for displaying a progress icon.